tests/php/View/SSViewerTest.php

```diff
@@ -2369,10 +2368,15 @@ public function testPrimitivesConvertedToDBFields()
         );
     }
 
+    #[DoesNotPerformAssertions]
     public function testMe(): void
     {
-        $mockArrayData = $this->getMockBuilder(ArrayData::class)->addMethods(['forTemplate'])->getMock();
```

Comment:
The addMethods() call now throws a deprecation warning, will be removed in phpunit12, so have rewritten test